posthog/api/person.py

```diff
@@ -174,6 +175,24 @@ def get_queryset(self):
         queryset = queryset.only("id", "created_at", "properties", "uuid", "is_identified")
         return queryset
 
+    def get_object(self):
+        queryset = self.filter_queryset(self.get_queryset())
+        person_id = self.kwargs[self.lookup_field]
+
+        try:
+            queryset = get_pk_or_uuid(queryset, person_id)
+        except ValueError:
+            raise ValidationError(
```

Comment:
Should this only be on certain actions?

E.g. if listing/retrieving then yes, but if updating it should be a 404?

Reply:
Hmmm not sure. If you are trying to call update but are passing in a dodgy looking ID I still think this error message is better than a 404 as we clearly know you're doing something wrong.